test/repositories.bzl

```diff
@@ -0,0 +1,20 @@
+"""Test dependencies for rules_java."""
+
+load("@bazel_skylib//lib:modules.bzl", "modules")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
```

Comment:
Any reason to not use http_jar from @rules_java itself? I think you said as much in https://github.com/bazelbuild/rules_java/pull/238

Reply:
Will update, that change just hadn't been merged yet and I didn't want to block this PR on it.

Reply:
Added a comment to explain why.

Reply:
It should be possible to use via the compatibility proxy layer (redirecting to `@bazel_tools` instead for older Bazel). But there's no need to block this PR on that and can happen later.